CMakeLists.txt

```diff
@@ -495,6 +495,84 @@ while(library_variants_copy)
             -DLIBCXX_ENABLE_WIDE_CHARACTERS=OFF
             -DLIBUNWIND_ENABLE_THREADS=OFF
         )
+
+        # picolibc tests
+        string(REPLACE " " ";" qemu_params "${qemu_params}")
+        add_custom_target(check-picolibc_${variant})
+        set(qemu_command "qemu-system-${cpu_family}")
+        # Tests disabled due to fact it fails at least at one variant
+        set(picolibc_tests
+            #abort # False negatve it returns non zero code which is expected
+            atexit
+            complex-funcs
+            constructor
+            #constructor-skip
+            fenv
+            ffs
+            hosted-exit
+            #long_double
+            #malloc
+            #malloc_stress
+            #math_errhandling
+            math-funcs
+            on_exit
+            posix-io
+            printf_scanf
+            printf-tests
+            rand
+            regex
+            setjmp
+            stack-smash
+            test-efcvt
+            #test-except
+            test-fopen
+            test-memset
+            test-mktemp
+            test-put
+            test-strchr
+            test-strtod
+            #timegm
+            time-sprintf
+            time-tests
+            #tls
+            ungetc
+        )
+
+        function(picolibc_test test)
+            # -Oz parameter is added just like picolibc is doing, otherwise there is a build issue
+            # ld.lld: error: undefined symbol: __issignalingl  while building math_errhandling test.
+            add_custom_command(
+                    OUTPUT check-picolibc_${variant}_build_${test}.out
```

Comment:
I think the output path is now incorrect

Reply:
Actually it is just a string that has to match one used in DEPENDS that is few lines below. I did test with completely dummy name and it worked. So functionally it seems to be correct. Let me know if you want me to change it.

Reply:
According to the docs, OUTPUT should "Specify the output files the command is expected to produce".
CMake has a habit of making things work in most circumstances, and not quite working in others, and I suspect this is one of those. For example, I wouldn't be surprised to find that the target gets rebuilt every time because an absent file is treated the same as it being out of date. So yes, change it please.

Reply:
Ok.